.github/workflows/cibuildwheel.yml

```
@@ -43,6 +53,7 @@ jobs:
           CIBW_MANYLINUX_I686_IMAGE: manylinux1
           CIBW_BEFORE_BUILD: pip install certifi numpy==1.19.3
           MPL_DISABLE_FH4: "yes"
+          CIBW_ARCHS_LINUX: ${{ matrix.cibw_archs }}
```

```suggestion
          CIBW_ARCHS: ${{ matrix.cibw_archs }}
```

I'd still recommend using ARCHS, since it will be much easier if/when you add universal2 or arm64 for macOS.